### PR TITLE
Add namespace flag option to "components", "configurations" and "list" CLI commands

### PR DESCRIPTION
**Pull request by marviniter**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
dapr list/components/configuration cmd add --namespace and --all-namespace flag for filter define namespace resource,and output list with namespace info.

### CODE REVIEW

**Comment by orizohar** [comment]
related to https://github.com/dapr/cli/issues/786
waiting on code PR https://github.com/dapr/cli/pull/787

File: daprdocs/content/en/reference/cli/dapr-components.md [hunk 1]
```diff
@@ -23,8 +23,12 @@ dapr components [flags]
 
 | Name | Environment Variable | Default | Description
 | --- | --- | --- | --- |
+| `--all-namespaces`, `-A` | | `false` | If true, list all Dapr components in all namespaces |
 | `--help`, `-h` | | | Print this help message |
 | `--kubernetes`, `-k` | | `false` | List all Dapr components in a Kubernetes cluster |
+| `--name` | |  | The components name to be printed (optional) |
+| `--namespace`, `-n` | | `default` | List define namespace components in a Kubernetes cluster |
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-components.md]: Need to be careful of saying "a Kubernetes cluster" here. Namespaces in Dapr are host independent. See https://docs.dapr.io/operations/components/component-scopes/. If this only applies to the -k option then that is a different thing and hence I would word this differently. Want to be clear here first on the intentions of this new flag.

Suggest this
| `--all-namespaces`, `-a` | | `false` | If true, list all components in all namespaces |
| `--namespace`, `-n` | | `default` | List all components in the specified namespace |

Also add more examples below that show these parameter. Like this
- dapr components -k -n production 
- dapr components -k -a

File: daprdocs/content/en/reference/cli/dapr-configurations.md [hunk 2]
```diff
@@ -23,8 +23,10 @@ dapr configurations [flags]
 
 | Name | Environment Variable | Default | Description
 | --- | --- | --- | --- |
+| `--all-namespaces`, `-A` | | `false` | If true, list all Dapr configurations in all namespaces
 | `--kubernetes`, `-k` | | `false` | List all Dapr configurations in a Kubernetes cluster
-| `--name`, `-n` | | | The configuration name to be printed (optional)
+| `--namespace`, `-n` | | `default` | List define namespace configurations in a Kubernetes cluster
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-configurations.md]: Same comment as above.

File: daprdocs/content/en/reference/cli/dapr-list.md [hunk 3]
```diff
@@ -23,8 +23,10 @@ dapr list [flags]
 
 | Name | Environment Variable | Default | Description
 | --- | --- | --- | --- |
+| `--all-namespaces`, `-A` | | `false` | List all Dapr pods in all namespaces |
 | `--help`, `-h` | | | Print this help message |
 | `--kubernetes`, `-k` | | `false` | List all Dapr pods in a Kubernetes cluster |
+| `--namespace`, `-n` | | `default` | List define namespace pods in a Kubernetes cluster |
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-list.md]: Same comment as above.

**Comment by greenie-msft** [comment]
@marviniter, can you also please change the target branch from v1.4 to v1.5? 

Thank you

**Comment by msfussell** [comment]
ping @marviniter - Can you take a look at these changes and also the code PR conflicts?

**Comment by msfussell** [comment]
For this PR also need to update the CLI ReadMe file overview as suggested in the code PR.

**Comment by mukundansundar** [comment]
@marviniter Will you be able to update the docs to the latest changes made in CLI?

**Comment by marviniter** [comment]
> @marviniter Will you be able to update the docs to the latest changes made in CLI?

@mukundansundar I will adjust the documentation description later

File: daprdocs/content/en/reference/cli/dapr-components.md (hunk 1 above)
Comment (mukundansundar) [reply daprdocs/content/en/reference/cli/dapr-components.md]: `-k` is a required flag @msfussell. So for now `dapr components` command only works on k8s cluster.

**Comment by msfussell** [comment]
@marviniter @mukundansundar - Need to get these docs updated in next couple of days for the v1.7 release. Is that possible?

**Comment by msfussell** [comment]
This needs attention.

File: daprdocs/content/en/reference/cli/dapr-configurations.md [hunk 4]
```diff
@@ -21,16 +21,28 @@ dapr configurations [flags]
...
-| -------------------- | -------------------- | ------- | ---------------------------------------------------- |
-| `--kubernetes`, `-k` |                      | `false` | List all Dapr configurations in a Kubernetes cluster |
-| `--name`, `-n`       |                      |         | The configuration name to be printed (optional)      |
-| `--output`, `-o`     |                      | `list`  | Output format (options: json or yaml or list)        |
-| `--help`, `-h`       |                      |         | Print this help message                              |
+
+| Name | Environment Variable | Default | Description
+| --- | --- | --- | --- |
+| `--kubernetes`, `-k` | | `false` | List all Dapr configurations in Kubernetes cluster (required).
+| `--all-namespaces`, `-A` | | `true` | If true, list all Dapr configurations in all namespaces (optional)
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-configurations.md:28]:
```suggestion
| `--all-namespaces`, `-A` | | `true` | If true, list all Dapr configurations in all namespaces (optional)
```

File: daprdocs/content/en/reference/cli/dapr-configurations.md [hunk 5]
```diff
@@ -21,16 +21,28 @@ dapr configurations [flags]
...
+| `--namespace` | | | List Dapr configurations in specific namespace.
+| `--name`, `-n` | | | Print specific Dapr configuration. (optional)
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-configurations.md:30]:
```suggestion
| `--name`, `-n` | | | Print specific Dapr configuration (optional)
```

File: daprdocs/content/en/reference/cli/dapr-configurations.md [hunk 6]
```diff
@@ -21,16 +21,28 @@ dapr configurations [flags]
...
-# List Kubernetes Dapr configurations
+# List Dapr configurations in all namespaces in Kubernetes mode
 dapr configurations -k
-```
+
+# List Dapr configurations in specific namespace in Kubernetes mode
+dapr configurations -k --namespace default
+
+# Print specific Dapr configuration in Kubernetes mode
+dapr configurations -k -n target
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-configurations.md]:
```suggestion
dapr configurations -k -n appconfig
```

File: daprdocs/content/en/reference/cli/dapr-components.md [hunk 7]
```diff
@@ -21,14 +21,28 @@ dapr components [flags]
...
-# List Kubernetes components
+# List Dapr components in all namespaces in Kubernetes mode
 dapr components -k
-```
+
+# List Dapr components in specific namespace in Kubernetes mode
+dapr components -k --namespace default
+
+# Print specific Dapr component in Kubernetes mode
+dapr components -k -n target
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-components.md]:
```suggestion
dapr components -k -n mycomponent
```

File: daprdocs/content/en/reference/cli/dapr-list.md [hunk 8]
```diff
@@ -22,21 +22,30 @@ dapr list [flags]
...
-| `--help`, `-h`       |                      |         | Print this help message                                                     |
-| `--kubernetes`, `-k` |                      | `false` | List all Dapr pods in a Kubernetes cluster                                  |
-| `--output`, `-o`     |                      | `table` | The output format of the list. Valid values are: `json`, `yaml`, or `table` |
+
+| Name | Environment Variable | Default | Description
+| --- | --- | --- | --- |
+| `--all-namespaces`, `-A` | | `false` | List all Dapr pods in all namespaces (optional) |
+| `--help`, `-h` | | | Print this help message |
+| `--kubernetes`, `-k` | | `false` | List all Dapr pods in a Kubernetes cluster (optional) |
+| `--namespace`, `-n` | | `default` | List define namespace pods in Kubernetes. Only with `-k` flag (optional) |
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-list.md]:
```suggestion
| `--namespace`, `-n` | | `default` | List the Dapr pods in the defined namespace in Kubernetes. Only with `-k` flag (optional) |
```

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**